binary_to_decimal/binary_to_decimal.js

```diff
@@ -0,0 +1,53 @@
+// ----------------------------------
+// Function binaryToDecimal
+
+function binaryToDecimal(binary_number){
+    const length = binary_number.toString().length; 
+    const string = binary_number.toString();
+    let binaryVal = true; //default value
+
+    for(let i = 0; i < length; i++){  //binary check - compare string
+        if(string[i] == "0" || string[i] == "1"){}
```

Comment:
Zkus se vyhýbat prázdným blokům, je to vlastně kód navíc. Ideální je tedy zapsat podmínku naopak, tzv:
```javascript
if (string[i] !== "0" || string[i] !== "1")
      binaryVal = false;
```
A vyhnu se `else` :)